<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PROTOTYPE BAC23 ALGO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
font-size: 20px;
color: blue;</string>
    </property>
    <property name="text">
     <string>Identifiant : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
font-size: 20px;
color: blue;</string>
    </property>
    <property name="text">
     <string>Numero De Telephone : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
font-size: 20px;
color: blue;</string>
    </property>
    <property name="text">
     <string>Genre : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
font-size: 20px;
color: blue;</string>
    </property>
    <property name="text">
     <string>Ville : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
font-size: 20px;
color: blue;</string>
    </property>
    <property name="text">
     <string>Etat :</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="inscrit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>186</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Inscription</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ville">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Tunis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Beja</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gafsa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bizerte</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="masculin">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Masculin</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="feminin">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Feminin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="identifiant">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="numtel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableau">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>521</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Identifiant</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NÂ°Telephone</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Genre</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ville</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Etat</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="ajouter">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="affiche_client">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Afficher Clients</string>
    </property>
   </widget>
   <widget class="QPushButton" name="affiche_chance">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Afficher Chances</string>
    </property>
   </widget>
   <widget class="QPushButton" name="affiche_gagnant">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Afficher Gagnants</string>
    </property>
   </widget>
   <widget class="QListWidget" name="chance_list">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="gagne_list">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>261</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
